### PR TITLE
Feature/vendor din sc

### PR DESCRIPTION
**Pull request by AusIV**
Update the registry integration branch to vendor din-sc, since its repository is currently private.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -2,7 +2,6 @@ module github.com/DIN-center/din-caddy-plugins
 
 go 1.22
 
-toolchain go1.22.8
 
 require (
 	github.com/DIN-center/din-sc/apps/din-go v0.0.0-20241029204756-63334786749d
```
Comment (natefikru) [inline go.mod:7]: what build version are you bringing in from din-sc, i can't tell. is it just latest? i see it as `v0.0.0-20241029204756-63334786749d`

Comment (natefikru) [reply go.mod:7]: im currently pondering if there is a way to do this automatically instead of having to statically bring the files over each time.

**Comment by AusIV** [comment]
So, it looks like vendoring is all or nothing within go. It didn't turn up in my test builds through caddy because Caddy treats the plugins as a package rather than the build target, so it's not concerned about the plugins' go.mod

In any case, I just renamed the "vendor" directory to "upstream" to get the tests to pass. The absence of a "vendor" directory made Go not worry about not being able to find other packages in the go.mod.

File: go.mod (hunk 1 above)
Comment (AusIV) [reply go.mod:7]: Based on the conversation with Tom and Amal, din-sc should be public the first week in january, so we won't have to worry about this for too long.

**Pull request merged**